client/components/Header.jsx

```diff
@@ -1,34 +1,40 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
 
-class LandingHeader extends React.Component {
-  render() {
-    return (
-      <header>
-        <div className="navbar-fixed">
-          <nav>
-            <div className="container-fluid">
-              <div className="nav-wrapper">
-                <a href="" className="brand-logo">ShoutIt</a>
-                <a href="" data-activates="mobile-links" className="button-collapse">
-                  <i className="material-icons">menu</i></a>
+const Header = props => (
+  <header>
+    <div className="navbar-fixed">
+      <nav>
+        <div className="container-fluid">
+          <div className="nav-wrapper">
+            <Link to="/" className="brand-logo">ShoutIt</Link>
+            <Link to="" data-activates="mobile-as" className="button-collapse">
+              <i className="material-icons">menu</i></Link>
 
-                <ul id="nav-mobile" className="right hide-on-med-and-down">
-                  <li className="active"><a href="">Home</a></li>
-                  <li><a href="#sign-in" className="modal-trigger">Sign in</a></li>
-                  <li><a href="#sign-up" className="modal-trigger">Get Started</a></li>
-                </ul>
-              </div>
-            </div>
-          </nav>
+            <ul id="nav-mobile" className="right hide-on-med-and-down">
+              <li><Link to={props.signInHref} className="modal-trigger">{props.signIn}</Link></li>
```

Comment:
You need to be more descriptive when naming variables, `signInHref` can be confusing.